### ISSUE TITLE
Increasing the the size of the main content in the docs

### ISSUE THREAD

**Issue by OmLanke** [Pycord-Development/pycord#1799]
### Summary

The main content size should be increased and sidebar sizes should be decreased

### What is the feature request for?

The documentation

### The Problem

In the new Furo theme based docs, the sidebars take up a lot of space, leaving the main content quite small. Although this may make it look consistent across screen sizes, it is a waste of screen real estate available. 

Currently how the docs look-
![Screenshot 2022-11-24 at 10 10 22 PM](https://user-images.githubusercontent.com/92863779/203833076-736285fa-6599-482f-af7c-8ddef3cb200d.png)

### The Ideal Solution

The sidebar and TOC are pushed to the sides and the main content occupies rest of the space available.

Like this-
![Screenshot 2022-11-24 at 10 13 18 PM](https://user-images.githubusercontent.com/92863779/203833618-6490a22c-7f7d-4ab7-9267-5599db7bc0b1.png)

### The Current Solution

Here is a video of how it can be done using the browser developer tools-

https://user-images.githubusercontent.com/92863779/203833790-5031ec85-6e6d-44d3-b392-1d4561fa90ae.mov

**Comment by Icebluewolf** [comment]
Bump, this would still be a good fix. 
I was able to just add the code below to `docs/_static/css/custom.css` and it seems to work.
```css
.sidebar-drawer {
  width: 0;
}

.content {
  width: fit-content;
}
```
The only downside is that the main content gets very spread out on very wide monitors. I did not test this change on my phone but it *should* have no effect. I have never worked with CSS before so if someone actually knows how to do this go ahead.

**Comment by Lulalaby** [comment]
We'll work on that soon

**Comment by Lulalaby** [comment]
_self ping_

**Comment by Paillat-dev** [comment]
I do not believe that we should be increasing the with of content to fit the screen width, if anything we could slightly increase it by something like 10-20%, for readability reasons:
https://journals.uc.edu/index.php/vl/article/view/5765/4629

**Comment by Paillat-dev** [comment]
> We'll work on that soon

Soon be like

**Comment by Lulalaby** [comment]
> Soon be like

Everything worked as intended.
I didn't lift a finger. You finished it :3
